financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryScreen.kt

```diff
@@ -199,22 +200,67 @@ private fun ManualEntryLoaded(
                     color = FinancialConnectionsTheme.colors.textCritical,
                 )
             }
-            Spacer(modifier = Modifier.weight(1f))
+        },
+        footer = {
+            ManualEntryFooter(
+                isValidForm = isValidForm,
+                loading = loading,
+                onSubmit = onSubmit
+            )
         }
-        // Footer
-        ManualEntryFooter(isValidForm, onSubmit)
-    }
+    )
+}
+
+@Composable
+private fun AccountForm(
```

Comment:
We should avoid returning multiple things in Composable methods. Can you wrap this in a column or so?